docs/SX126x_SX127x_INCOMPATIBILITY.md

```diff
@@ -0,0 +1,13 @@
+# mLRS Documentation: SX126x and SX127x Incompatibility #
+
+([back to main page](../README.md))
+
+mLRS hardware using the sx126x and sx127x LoRa chipsets are not compatible with each other, and cannot be operated together (they don't bind).
```

Comment:
Perhaps 'mLRS hardware using the SX126x and SX127x LoRa chipsets are **not** compatible with each other and will not bind.'

I prefer SX in caps to match the datasheet.

Reply:
done